src/lib/helpers/date.ts

```diff
@@ -73,6 +82,52 @@ export const toLocaleTimeISO = (datetime: string | number) => {
     return date.toLocaleTimeString('sv');
 };
 
+export const utcHourToLocaleHour = (utcTimeString: string) => {
+    const now = new Date();
+    const [hours, minutes] = utcTimeString.split(':').map(Number);
+    const customTime = Date.UTC(now.getFullYear(), now.getMonth(), now.getDate(), hours, minutes);
+
+    const options: Intl.DateTimeFormatOptions = {
+        hour: 'numeric',
+        minute: 'numeric',
+        hourCycle: 'h23'
+    };
+
+    return new Intl.DateTimeFormat('en', options).format(new Date(customTime));
+};
+
+export const utcWeekDayToLocaleWeekDay = (utcDay: WeekDay, utcTimeString: string) => {
+    const now = new Date();
+
+    const daysOfTheWeek: WeekDay[] = [
+        'Sunday',
+        'Monday',
+        'Tuesday',
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday'
+    ];
+
+    const utcDayNumber = daysOfTheWeek.indexOf(utcDay);
```

Comment:
Using an object like you did before is better than using an array like this because the object is O(1) lookup vs O(n) using this array.

Reply:
Done.